trompace/__init__.py

```diff
@@ -64,7 +66,10 @@ def make_parameters(**kwargs):
         elif isinstance(v, datetime.datetime):
             value = f"{{formatted: {encoder.encode(v.isoformat())}}}"
         elif isinstance(v, list):
+            #value = "[{}]".format(", ".join(item for item in encode_list(v, encoder)))
```

Comment:
This looks like it's not needed

Reply:
No I think it is needed because a field value can be a list of dictionaries